extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/experience/customerJourneyManagement/messageprofile",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe CJM ExperienceEvent - MessageProfile Details",
+  "description": "MessageProfile details for the Adobe CJM ExperienceEvent.",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+
+  "definitions": {
+    "messageProfile": {
+      "properties": {
+        "@id": {
+          "title": "MessageProfile ID",
+          "type": "string",
+          "description": "A unique id refers to every single message sent as part of a MessageExecution activity."
+        },
+        "xdm:channel": {
+          "title": "Experience channel",
+          "description": "Experience channel related to the ExperienceEvent.",
+          "$ref": "https://ns.adobe.com/xdm/channels/channel"
+        },
+        "xdm:variant": {
+          "title": "Content Variant",
```

Comment:
Can variant be more complex and nested?

What if message header had 4 variants, message body had 2, and message footer had 3.

Is that possible or would variants be a cumulative sum of outcomes?

Reply:
I am not aware of this requirement, if we have this requirement then we should add a support for defining multiple variants with respective placement.

Reply:
As far as I also know there is a variant at a message profile level, complex nesting is there. @jfait Are you aware of any such possible requirements?